amplipi/streams.py

```diff
@@ -86,6 +86,12 @@ def uuid_gen():
   # Generic UUID in case of failure
   return '39ae35cc-b4c1-444d-b13a-294898d771fa'
 
+class StreamError(Exception):
```

Comment:
Would this work better as an InvalidField exception? That way the code could always print the message with the same formatting.